src/app/services/payment-service.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { PaymentService } from './payment-service';
+import { PaymentService } from './payment/payment-service';
```

Comment:
@patrickf949 If you are moving the service file to a different folder, ensure to move it's spec file there too and remove it from here.